### DISCUSSION TITLE
pitch doesn't work if followUserLocation equal true

### DISCUSSION THREAD

**Discussion by buzlan** [rnmapbox/maps#2861] [Q&A]
I tried to use the followPitch property, but I encountered freezing issues when updating the speed.
When I use the pitch prop with followUserLocation set to false, everything works well. However, I need to adjust the pitch when following the user's location.

This is my camera component:

 <Camera
          animationMode='linearTo'
          animationDuration={1500}
          followUserLocation={followUser}
          maxZoomLevel={Config.map.maxZoom}
          // pitch={Number(speed) * 3.5 > 0 ? Number(Number(speed * 3.5).toFixed(0)) : 0} doesn't work when followUserLocation={true}
          followUserMode={'course'}
        />

**Comment by DimaVakula** [comment]
+1

**Comment by fedarenchyk** [comment]
+1
